echo-scheduler/src/main/groovy/com/netflix/spinnaker/echo/scheduler/actions/pipeline/PipelineConfigsPollingJob.groovy

```diff
@@ -0,0 +1,186 @@
+/*
```

Comment:
New classes should be java.

Reply:
this a move + major edit of old `PipelineConfigsPollingAgent.groovy`. but I can convert to Java in a separate PR